README.md

```
@@ -83,13 +84,14 @@ order := sveawebpay.RecurOrder{
     Vat: 25,
 }
 
-transaction, err := c.Recur(order)
+paymentResponse, err := c.Recur(order)
+transaction := paymentResponse.Transaction
 ```
 
 ## Cancel a recur subscription
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)